images/toolchain-arm-bcm2708hardfp-linux-gnueabi/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:stable
 
+ARG MAJOR
```

Comment:
Why did you add `MAJOR` here? As far as I can see, it's not used in this dockerfile. For `toolchain-arm-bcm2708hardfp-linux-gnueabi`, we should specify `COMMIT` instead.